src/tests/sorting.spec.js

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '../fixtures/base';
+
+test.describe('sorting invertory page', () => {
+    test('sorting name A-Z', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('az');
+
+        await expect(app.inventory.inventoryItems.nth(0).getByTestId('inventory-item-name')).toHaveText('Sauce Labs Backpack');
```

Comment:
You'll need to fix this test everytime products added
I'd suggest to 
1. Grab all products info including name/price/description whatever
2. Select sorting option 
3. Sort the collected data by yourself (to have the expected order)
4. Grab all products info from the page after sorting 
5. Compare expected order with actual